### PR TITLE
Finish 3rd task(UI)

### PR DESCRIPTION
**Pull request by AlexeSemenenko**

### CODE REVIEW

File: UI/styles.css [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+    background-color: #d1d1d1;
+    z-index: 1;
+    height: 6.5%;
+    width: 1000%;
+    top: 0%;
+    left: 0%;
+}
+
+.site-name {
+    position: absolute;
```
Comment (MaksimViarenich) [inline UI/styles.css]: Don't use absolute everywhere) Try to work with relative

File: UI/styles.css [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+}
+
+.post10 {
+    position: absolute;
+    background-color: white;
+    width: 90%;
+    height: 20%;
+    left: 5%;
+    top: 233%;
+}
```
Comment (MaksimViarenich) [inline UI/styles.css:147]: What will happen with post11?)

**Comment by MaksimViarenich** [comment]
Try to change the height and you will see this)
image
And height
image

The reason of it - **position: absolute;**

File: UI/index.html [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                <img class="like-img" src="like.png">
+
+                <p class="like-counter">322</p>
+
+                <span class="hashtags-place">
+                    #helloworld #like4like #follow4follow #first
+                </span>
+            </div>
+
+            <div class="post8">
```
Comment (MaksimViarenich) [inline UI/index.html]: It's very very very very very very ... very bad. Your HTML shouldn't depends on the number of posts.

**Comment by MaksimViarenich** [comment]
We should discuss everything in the lesson

**Comment by AlexeSemenenko** [comment]
> We should discuss everything in the lesson

Good evening, Maxim! Fixed all bugs, please check the new version.

**Review by MaksimViarenich — APPROVED** [review]

File: UI/index.html (hunk 3 above)
Comment (AlexeSemenenko) [reply UI/index.html]: Yes, well, I realized that initially there were a lot of mistakes. But now I'm a little confused about whether I can merge pull request?

**Pull request merged**